frontend/lib/ui/navbar.tsx

```diff
@@ -48,6 +59,15 @@ const NavbarContext = React.createContext<NavbarContext>({
   toggleDropdown: () => {},
 });
 
+export function getUserInitials({
+  firstName,
+  lastName,
+}: Pick<AllSessionInfo, "firstName" | "lastName">): string {
+  return [firstName, lastName]
+    .map((value) => (value && value[0].toUpperCase()) || "?")
```

Comment:
Er, so this will replace any non-specified user initial with a "?", which I think is okay for now since we require both first and last name when users onboard--so the question marks should never really show, and are intended only to make sure this menu is completely empty in extremely rare edge cases (e.g. if the account is created via `manage.py createsuperuser`).

However, if we move to a "preferred name" system that may make one of the first/last name fields optional or something, we will need to revisit this.

Reply:
Oh gotcha! Yeah I mean since this should never really show up, I don't see any _huge_ need to spend time on this, but I will be honest that seeing a question mark surrounded by a circle may get confusing as we use similar symbols for things like helper text. My preference would be to use something like the [UTF-8 Gear icon](https://www.utf8icons.com/character/9881/gear), or even a plus sign, instead of the question mark, but I'll leave that decision up to you.

Reply:
Makes sense to me!

Reply:
Wait were you originally suggesting using a gear icon here instead of a question mark?  I saw that in the email version of this notification and thought it was a good idea so I made that change!